RLBotPack/Skybot/SkybotPreset.cfg

```diff
@@ -0,0 +1,11 @@
+[Locations]

```

Comment:
You've caused Skybot to appear twice in the bot list by including this .cfg file, can you please remove it?